warrior/WarriorCore/testsuite_driver.py

```diff
@@ -335,14 +335,7 @@ def execute_testsuite(testsuite_filepath, data_repository, from_project,
                 break
     
     elif execution_type.upper() == 'RUN_MULTIPLE':
-        Max_Attempts = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Max_Attempts')
-        Number_Attempts = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Number_Attempts')
-        
-        if Max_Attempts == "":
-            execution_value = Number_Attempts
-        else:
-            execution_value = Max_Attempts
-        
+        execution_value = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Number_Attempts')
```

Comment:
@brettleehari 

- #1: we should go with both for backward comatibility, i dont think we can get rid of one at this moment.

- Agree that the error handling should be applied for all the cases like rup, ruf, rmt. 
Where should we do it? data validation  should be done at the front end for sure.. but we should not totally ignore error handling at warrior level. Because even today there are lot of users who use warrior directly without using the UI. 
I would suggest to raise a ticket for error handling at UI, and handle errors in warrior using this ticket.
If we encounter an un supported value then we should raise a warning that prints out what was expected and what was received and that default value will be used instead with the default value printed.